scout/constants/variant_tags.py

```diff
@@ -96,6 +96,15 @@
     ("XD_dn", "X Linked Dominant De Novo"),
 )
 
+# Used for filter variants in blueprints.variants
+VARIANT_GENOTYPES = (
+    ("", ""),
+    ("0/1", "Heterozygous (0/1)"),
+    ("1/1", "Homozygous (1/1)"),
+    ("0/0", "Homozygous ref (0/0)"),
+    ("./.", "Allele missing (./.)"),
```

Comment:
"Allele missing" is ambiguous. It could also mean one allele missing, in which case the pattern could be any combination with a "." in it.

Reply:
Yes, but in this case I mean both alleles missing. I'll write like this!